scripts/pycairoinstall.py

```diff
@@ -4,48 +4,48 @@
 import urllib.request
 
 if 'Windows' in platform.system():
-    # In case the python version is 3.6 and the system is 32-bit, try pycairo‑1.19.1‑cp37‑cp37m‑win32.whl version of cairo
+    # In case the python version is 3.6 and the system is 32-bit, try pycairo‑1.19.2‑cp37‑cp37m‑win32.whl version of cairo
     if sys.version[:3]=='3.6' and platform.machine()=='x86':
-        urllib.request.urlretrieve("https://download.lfd.uci.edu/pythonlibs/w3jqiv8s/pycairo-1.19.1-cp36-cp36m-win32.whl", "pycairo-1.19.1-cp36-cp36m-win32.whl")
-        os.system("pip install pycairo-1.19.1-cp36-cp36m-win32.whl")
-        os.remove("pycairo-1.19.1-cp37-cp37m-win32.whl")
+        urllib.request.urlretrieve("https://github.com/naveen521kk/pycairo/releases/download/v1.19.2/pycairo-1.19.2-cp36-cp36m-win32.whl", "pycairo-1.19.2-cp36-cp36m-win32.whl")
+        os.system("pip3.6 install pycairo-1.19.2-cp36-cp36m-win32.whl")
```

Comment:
**Issue (non-blocking)** : I'm a little bit worried about this line. Is there a particular reason you use `pip3.6 over pip ?`
`pip3.6` won't work, when someone use `pip` or `pip3 `instead of `pip3.6`
Do you think you could replace it with `python -m pip` ? Would it work ?

Reply:
It will work perfectly on windows.